Terminal-iOS/MyStudy/StudyDetail/ViewComponent/StudyDetail/StudyDetailInteractor.swift

```diff
@@ -20,6 +20,15 @@ final class StudyDetailInteractor: StudyDetailInteractorInputProtocol {
     func postStudyJoin(studyID: Int, message: String) {
         remoteDatamanager?.postStudyJoin(studyID: studyID, message: message)
     }
+    
+    func postReportStudy(studyID: Int, reportMessage: String) {
+        if reportMessage == "허위 신고 시 이용이 제한될 수 있습니다." ||
+            reportMessage.isEmpty {
```

Comment:
```suggestion
        if reportMessage == "허위 신고 시 이용이 제한될 수 있습니다."
           ||  reportMessage.isEmpty {
```
가독성을 위해서 이렇게 하는게 좋겠어요